server/src/test/java/com/cloud/storage/VolumeApiServiceImplTest.java

```diff
@@ -1164,6 +1166,65 @@ public void doesTargetStorageSupportDiskOfferingTestDiskOfferingTagsEqualsStorag
         Assert.assertTrue(result);
     }
 
+    @Test
+    public void validateIfVmHaveBackupsTestExceptionWhenTryToDetachVolumeFromVMWhichBackupOffering() {
```

Comment:
```suggestion
    public void validateIfVmHaveBackupsTestExceptionWhenTryToDetachVolumeFromVmWithBackupOffering() {
```
?
It sounds strange as it is written now